tests/integration_test.go

```diff
@@ -30,11 +29,9 @@ func testIntegration(t *testing.T, funcs ...func(t *testing.T, sb integration.Sa
 	mirroredImages := integration.OfficialImages("busybox:latest", "alpine:latest")
 	buildkitImage := "docker.io/moby/buildkit:buildx-stable-1"
 	if integration.IsTestDockerd() {
-		if img, ok := os.LookupEnv("TEST_BUILDKIT_IMAGE"); ok {
-			ref, err := reference.ParseNormalizedNamed(img)
-			if err == nil {
-				buildkitImage = ref.String()
-			}
+		if img, ok := os.LookupEnv("TEST_BUILDKIT_IMAGE"); ok && img != "" {
+			t.Log("using TEST_BUILDKIT_IMAGE", img)
```

Comment:
Not sure if needed, but thought to make it more visible if a custom image is used, in that odd case an invalid image reference is passed.